tests/plugins/autogrow/_helpers/tools.js

```diff
@@ -19,8 +19,36 @@ var autogrowTools = ( function() {
 		return html;
 	}
 
+	function testEditorSizeWithContent( editorWidth ) {
+		bender.editorBot.create( {
+			name: 'EditorWidth' + editorWidth,
```

Comment:
Running this test twice with the same `editorWidth` will result in an error as we will try to create two editors with the same name. Since this test helper has been extracted as a separate tool method, maybe we could improve it by adding some more random editor name? As an example. you could use `new Date().getTime()` as it's really unlikely it will match milliseconds:

```js
name: 'editor_' + new Date().getTime()
```